### PR TITLE
`--repeat` flag added to MyTimer

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#94 

#### What does this implement/fix? Explain your changes.
This PR will add `--repeat` flag to MyTimer so that user can play MyTimer in a loop.
Using `--repeat -1` will run the program in an infinite loop.

#### Any other comments?
@sepandhaghighi please also add the new milestone before merging this PR.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `--repeat` flag added
```
Comment (sepandhaghighi) [inline CHANGELOG.md:9]: Remove added

File: README.md [hunk 2]
```diff
@@ -161,7 +161,13 @@ mytimer --program=black-tea
 ```console
 mytimer --minute=7 --second=30 --message="Test message"
 ```
-		
+
+### Repeat
+
+```console
+mytimer --repeat 3 --second 4
```
Comment (sepandhaghighi) [inline README.md:168]: `mytimer --repeat=3 --second=4`

File: README.md [hunk 3]
```diff
@@ -161,7 +161,13 @@ mytimer --program=black-tea
...
+```
+* Use `--repeat -1` for infinite loop
```
Comment (sepandhaghighi) [inline README.md:170]: `--repeat=-1`

File: mytimer/functions.py [hunk 4]
```diff
@@ -459,16 +459,28 @@ def run_timer(args):
...
+                    params=params,
+                    long_break_params=long_break_params,
+                    short_break_params=short_break_params)
+            elif args.program in ["52-17", "112-26", "animedoro"]:
+                break_params = load_params(args, is_break=True)
+                two_step_timer(timer_func, params1=params, params2=break_params)
+            else:
+                timer_func(**params)
+
+            end_round_message = END_ROUND_MESSAGE.format(f'{timer_round + 1}/{args.repeat}')
```
Comment (sepandhaghighi) [inline mytimer/functions.py:479]: I suggest using `format` instead of `f`

File: mytimer/functions.py [hunk 5]
```diff
@@ -459,16 +459,28 @@ def run_timer(args):
...
+            if args.repeat == -1:
+                end_round_message = END_ROUND_MESSAGE.format(f'{timer_round + 1}')
```
Comment (sepandhaghighi) [inline mytimer/functions.py:481]: I suggest using `format` instead of `f`

File: mytimer/functions.py [hunk 6]
```diff
@@ -459,16 +459,28 @@ def run_timer(args):
...
+            print_message(
```
Comment (sepandhaghighi) [inline mytimer/functions.py:482]: I suggest we skip this line for the final round of the loop. Do you agree?

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Thanks for your efforts 🔥 
Take a look at my comments 💯

File: mytimer/functions.py (hunk 6 above)
Comment (sadrasabouri) [reply mytimer/functions.py]: Fixed in [fa7a0c4](https://github.com/sepandhaghighi/mytimer/pull/95/commits/fa7a0c4deff22df0b6ec8b6088373a3a54d340d6).

**Comment by sadrasabouri** [comment]
@sepandhaghighi please re-review.

File: mytimer/functions.py [hunk 7]
```diff
@@ -459,16 +459,29 @@ def run_timer(args):
...
+            elif args.program in ["52-17", "112-26", "animedoro"]:
+                break_params = load_params(args, is_break=True)
+                two_step_timer(timer_func, params1=params, params2=break_params)
+            else:
+                timer_func(**params)
+
+            end_round_message = END_ROUND_MESSAGE.format("{}/{}".format(timer_round + 1, args.repeat))
+            if args.repeat == -1:
+                end_round_message = END_ROUND_MESSAGE.format(timer_round + 1)
+            if timer_round < args.repeat - 1 or args.repeat == -1:
```
Comment (sepandhaghighi) [inline mytimer/functions.py:482]: Why `or args.repeat == -1`?
Under infinite loop mode,`end_round_message` will never be printed.

File: CHANGELOG.md [hunk 8]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `--repeat` flag
```
Comment (sepandhaghighi) [inline CHANGELOG.md:9]: flag --> argument

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Take a look at my comments 💯

File: README.md [hunk 9]
```diff
@@ -161,7 +161,13 @@ mytimer --program=black-tea
 ```console
 mytimer --minute=7 --second=30 --message="Test message"
 ```
-		
+
+### Repeat
+
```
Comment (sepandhaghighi) [inline README.md:166]: ⚠️ The default value is `1`

File: mytimer/functions.py (hunk 7 above)
Comment (sadrasabouri) [reply mytimer/functions.py:482]: OK. I will fix it then.

**Review by sepandhaghighi — APPROVED** [review]
@sadrasabouri Well done 🔥

**Pull request merged**